host_core/lib/host_core/wasmcloud/native.ex

```diff
@@ -35,6 +35,7 @@ defmodule HostCore.WasmCloud.Native do
   def get_provider_bindle(_creds, _bindle_id, _link_name), do: error()
 
   # Wasm Runtime
+  def runtime_init(), do: error()
```

Comment:
The runtime is instantiated by the virtual host (`HostCore.Vhost.VirtualHost`) process. The architecture is that there is one instance of the rust runtime (aka `libwasmcloud`) per virtual host. There should be no global initialization at all. 

I suggest performing this initialization during the instantiation of the runtime by the virtual host. If there are configuration parameters that need to be passed in, you can add them to the `Runtime.Configuration` struct that currently only has one field.

Reply:
Will `runtime_new` be called at most once per thread?